Yoast/javascript
# Bundle CSS from components package

## Summary


This PR can be summarized in the following changelog entry:

* [components]CSS is now imported in JavaScript files, and will therefore be present in the package.

## Relevant technical choices:

* See issue #517 

## Test instructions

This PR can be tested by following these steps:

* Go to the components example app >>> button page, inspect the HTML, add `class="yoast"` to the `<body>`, and replace the classes in one of the buttons with `class="yoast-button yoast-button--primary"`. The button should be styled.
* Also test in conjunction with PR on `wordpress-seo`.
https://github.com/Yoast/wordpress-seo/pull/14548

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * CSS is now exported in the JS package, which has the potentional for causing problems. We should probably take extra care that our JS is loading everywhere.

## UI changes

## Quality assurance


Fixes #517
